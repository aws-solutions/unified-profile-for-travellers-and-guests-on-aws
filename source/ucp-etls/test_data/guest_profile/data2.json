{    "modelVersion": "1.0",    "id": "9638726649",    "lastUpdatedOn": "2023-03-15T23:55:22.375409789Z",    "createdOn": "2022-12-15T23:55:22.375409789Z",    "lastUpdatedBy": "Ayla Hagenes",    "createdBy": "Griffin Heidenreich",    "emails": [        {            "type": "",            "address": "zellagreenfelder@koepp.biz",            "primary": true        },        {            "type": "personal",            "address": "helenelehner@conroy.io",            "primary": false        }    ],    "phones": [        {            "type": "home",            "number": "9040863796",            "primary": true,            "countryCode": 93        },        {            "type": "home",            "number": "713.378.3100",            "primary": false,            "countryCode": 61        }    ],    "addresses": [        {            "type": "business",            "line1": "12739 East Hollow mouth",            "line2": "more content line 2",            "line3": "more content line 3",            "line4": "more content line 4",            "city": "Reno",            "stateProvince": {                "code": "MS",                "name": ""            },            "postalCode": "92405",            "country": {                "code": "DZ",                "name": ""            },            "primary": true        }    ],    "honorific": "",    "firstName": "Emilia",    "middleName": "Warren",    "lastName": "McLaughlin",    "gender": "female",    "pronoun": "",    "dateOfBirth": "2020-05-11",    "language": {        "code": "it",        "name": ""    },    "nationality": {        "code": "FR",        "name": ""    },    "jobTitle": "Coordinator",    "parentCompany": "Impact Forecasting (Aon)",    "loyaltyPrograms": [        {            "id": "9179268077",            "programName": "reward",            "points": 356249.0,            "pointUnit": "points",            "pointsToNextLevel": 5259.0,            "level": "platinium",            "joined": "2022-12-15T23:55:22.375409789Z"        }    ]}