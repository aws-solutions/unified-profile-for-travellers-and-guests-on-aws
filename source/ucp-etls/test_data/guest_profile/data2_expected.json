{
    "common_email_recs": [
        {
            "address": "lelarowe@mclaughlin.com",
            "last_updated": "2022-03-24T23:07:59.386436665Z",
            "last_updated_by": "Abbey Little",
            "model_version": "1.0",
            "object_type": "email_history",
            "traveller_id": "8586451052",
            "type": "personal"
        }
    ],
    "common_phone_recs": [
        {
            "country_code": 53,
            "last_updated": "2022-03-24T23:07:59.386436665Z",
            "last_updated_by": "Abbey Little",
            "model_version": "1.0",
            "number": "4626146981",
            "object_type": "phone_history",
            "traveller_id": "8586451052",
            "type": "home"
        },
        {
            "country_code": 47,
            "last_updated": "2022-03-24T23:07:59.386436665Z",
            "last_updated_by": "Abbey Little",
            "model_version": "1.0",
            "number": "9465488670",
            "object_type": "phone_history",
            "traveller_id": "8586451052",
            "type": "personal"
        }
    ],
    "guest_profile_recs": [
        {
            "address_city": "",
            "address_country": "",
            "address_line1": "",
            "address_line2": "",
            "address_line3": "",
            "address_line4": "",
            "address_postal_code": "",
            "address_state_province": "",
            "address_type": "",
            "company": "PlotWatt",
            "created_by": "Madyson Mueller",
            "created_on": "2022-01-24T23:07:59.386436665Z",
            "date_of_birth": "1965-01-13",
            "email": "lelarowe@mclaughlin.com",
            "email_type": "personal",
            "first_name": "Timmothy",
            "gender": "other",
            "honorific": "Sir",
            "job_title": "Facilitator",
            "language_code": "ln",
            "language_name": "",
            "last_name": "Wiza",
            "last_updated_by": "Abbey Little",
            "last_updated_on": "2022-03-24T23:07:59.386436665Z",
            "middle_name": "Eliane",
            "model_version": "1.0",
            "nationality_code": "LU",
            "nationality_name": "",
            "object_type": "pax_profile",
            "phone_home": "4626146981",
            "phone_type": "home",
            "pronoun": "he",
            "traveller_id": "8586451052"
        }
    ],
    "hotel_loyalty_recs": [
        {
            "id": "7767442999",
            "joined": "2022-01-24T23:07:59.386436665Z",
            "last_updated": "2022-03-24T23:07:59.386436665Z",
            "last_updated_by": "Abbey Little",
            "level": "silver",
            "model_version": "1.0",
            "object_type": "air_loyalty",
            "points": 205918.0,
            "points_to_next_level": 8302.0,
            "program_name": "club",
            "traveller_id": "8586451052",
            "units": "miles"
        }
    ]
}